Documentation/policy/language.rst

```diff
@@ -34,6 +34,11 @@ can talk to each other. Layer 3 policies can be specified using the following me
   IP addresses or subnets into the policies. This construct should be used as a
   last resort as it requires stable IP or subnet assignments.
 
+* `DNS based`: Selects remote, non-cluster, peers using DNS names converted to
+  IPs via DNS lookups. It shares all limitations of the `CIDR based` rules
```

Comment:
CIDR-based

Reply:
I'd need to change the corresponding section header otherwise it won't link (I think). So, if I add the hyphen here, I'll have to change the whole list. Is that ok?

Reply:
@ianvernon Did you want the hyphen fixup as part of this PR or as another? If I do one I'll have to do all

Reply:
Sorry, I missed your replies. This isn't blocking, it's definitely a nitpick. We can get it fixed later.

Reply:
in the next PR, in fact ;) I just didn't want to queue up one based on this one